thin_client.py

```diff
@@ -153,6 +153,8 @@ def initializePygame(FPS):
     screen.blit(mouseLabel, (renderPosX - 100, renderPosY + 50))
     pygame.display.update()
     
+    return True
```

Comment:
you shouldn't return `True` just for the sake of return value without any meaning. I believed you need it for testing purposes. 

To solve this in the unit test, you can simply run the function, and handle the error by calling [`fail(msg)`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.fail) method. See example in my comment below.